README.md

```diff
@@ -36,6 +36,16 @@ PGHOST="POSTGRES_HOST"
 PGPASSWORD="POSTGRES_PASSWORD"
 PGDATABASE="POSTGRES_DATABASE"
 ```
+To work with the authentication functionalities, you need to add the following entries to the `.env` or `.env.local`:
+```
+NEXTAUTH_URL="HOST_URL"
```

Comment:
@EkkuJ this variable does not seem to be used anywhere else in the code. Is it needed, or where is it supposed to be needed?

Reply:
Nevermind, now I understand it is required by NextAuth